Docker/scripts/entrypoint.sh

```diff
@@ -31,8 +31,7 @@ fi
 # Run Merly Mentor Daemon and prefix logs with "DAEMON"
 ./MerlyMentor -N daemon --stdout &
 
-# Run Merly Mentor Bridge and prefix logs with "BRIDGE"
-./MentorBridge &
+
```

Comment:
@nimatel - minor nit on the extra spaces in this file, otherwise it looks great!